uiowa/uiowa
# Iowa Now experts list

Resolves https://github.com/uiowa/uiowa/issues/5670

# How to test

- `ddev blt ds --site=now.uiowa.edu`
- Add a research area to https://now.uiowa.ddev.site/admin/structure/taxonomy/manage/research_areas/overview
- Add a person profile with Title/position, email, phone, research areas and add the tag "expert"
- Go to https://now.uiowa.ddev.site/experts-list and verify that the above fields are listed in a card teaser format
- Try searching for something that is unavailable and check out the error message.
